lmod_proxy/edx_grades/actions.py

```diff
@@ -25,7 +25,9 @@ def post_grades(gradebook, form):
     approve_grades = False
     if current_app.config['LMODP_APPROVE_GRADES']:
         approve_grades = True
-    csv_file = TextIOWrapper(form.datafile.data.stream, encoding='utf8')
+    csv_file = tempfile.TemporaryFile(mode='w+')
+    csv_file.write(form.datafile.data.read().decode('utf8'))
```

Comment:
Why the switch away from TextIOWrapper?

Reply:
Because the underlying object is a SpooledTemporraryFile and not a BytesIO object, which means that the wrapper throws an exception about the object not having an attribute of `readable`. This was an easy workaround to handle converting the data to text.